src/univariates.jl

```diff
@@ -465,20 +465,20 @@ for fun in [:pdf, :logpdf,
 
     @eval begin
         function ($_fun!)(r::AbstractArray, d::UnivariateDistribution, X::AbstractArray)
-            for i in 1 : length(X)
+            for i in linearindices(X)
```

Comment:
I was simply following the advice at https://docs.julialang.org/en/latest/devdocs/offset-arrays/ which says

> replace `1:length(A)` with `linearindices(A)`